.github/workflows/cicd.yaml

```diff
@@ -160,3 +216,23 @@ jobs:
             openmldb-*.tar.gz
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  macos-release:
```

Comment:
foroget to see this. there is no need to add a new `macos-release` job here. 
the only thing needed for release job is update dependency in line https://github.com/4paradigm/OpenMLDB/pull/2028/files#diff-2c9df400afaffa9270cab617792f08cc671e5f95ae192f1cf111ef4e990a4addR203. 

someting like this:
```
needs: [ "cpp", "macos-cpp" ]
```

Reply:
Done. Thank you for your patient guidance